fabric-registry-sync-v0/src/testmod/java/net/fabricmc/fabric/test/registry/sync/RegistrySyncTest.java

```diff
@@ -109,6 +109,15 @@ public void onInitialize() {
 		Validate.isTrue(RegistryAttributeHolder.get(fabricRegistry).hasAttribute(RegistryAttribute.SYNCED));
 		Validate.isTrue(!RegistryAttributeHolder.get(fabricRegistry).hasAttribute(RegistryAttribute.PERSISTED));
 
+		SimpleRegistry<String> fakeVanillaRegistry = FabricRegistryBuilder.createSimple(String.class, new Identifier("registry_sync", "fake_vanilla_registry"))
+				.attribute(RegistryAttribute.SYNCED)
+				.buildAndRegister();
+
+		Registry.register(fakeVanillaRegistry, new Identifier("minecraft", "test"), "test");
+		Registry.register(fakeVanillaRegistry, new Identifier("brigadier", "test"), "test");
+
+		Validate.isTrue(!RegistryAttributeHolder.get(fakeVanillaRegistry).hasAttribute(RegistryAttribute.MODDED));
```

Comment:
This test wont work, as all registries created through the builder are marked as modded. I cannot think of a good way to "unit" test this. I will fix and manuall test. 👍